CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+### citus-docker v9.5.0.docker (November 12, 2020) ###
```

Comment:
Maybe we should add some entries for missing topn and hll packages in this image. Users could run `CREATE EXTENSION hll` previously, and now they cannot do this. I think this should be documented somewhere

Reply:
addressed c5f01bd